src/image-handler.ts

```diff
@@ -56,16 +56,18 @@ export class ImageHandler {
     // Only proceed if a cluster was selected
     if (selectedCluster) {
       let name = image.name;
-      let filename: string;
-      const env = { ...process.env };
+      let filename: string | undefined;
 
       // Create a name:tag string for the image
       if (image.tag) {
         name = name + ':' + image.tag;
       }
 
-      env.PATH = getMinikubePath();
-      env.MINIKUBE_HOME = getMinikubeHome();
+      const env: Record<string, string> = {
```

Comment:
issue: { ... process.env} is removed

Reply:
See https://github.com/podman-desktop/extension-minikube/pull/223#discussion_r1838046120